packages/app-check/src/api.test.ts

```diff
@@ -10,7 +10,7 @@
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or ied.
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
generally, lawyers get annoyed by changes to licenses...

A peer suggested that someone did a global search and replace `s/impl//`. -- I believe they're correct.